### PR TITLE
Add notes for CG 2023-09-12

### PR DESCRIPTION
**Pull request by dschuff**

### CODE REVIEW

**Comment by dschuff** [comment]
Sorry I just realized I hadn't posted these notes.

File: main/2023/CG-09-12.md [hunk 1]
```diff
@@ -40,4 +40,169 @@ Installation is required, see the calendar invite.
...
+Introducing BT as co-champion
+
+Couple of issues are linked in the EH proposal to reintroduce Exnref
+
+Will be voting on the proposed changes (reintroducing exnref) in Oct in-person CG
+
+
+#### Vote to advance function references and GC proposals to phase 4 [30 min]
+
+AR presenting (TODO: slides)
```
Comment (dschuff) [inline main/2023/CG-09-12.md:118]: @rossberg TODO: GC phase 4 slides

File: main/2023/CG-09-12.md [hunk 2]
```diff
@@ -40,4 +40,169 @@ Installation is required, see the calendar invite.
...
+
+DG: my understanding is that this isn’t a conditional vote, i think there was enough consensus that the changes are far enough along that people are comfortable that the missing pieces will be resolved soon.
+
+CW: I’m fine with interpreting it that way, still curious on the state of SIMD, though it’s less related to this discussion.
+
+DG: We wanted a full discussion on profiles before we do a “final” phase 4 vote. So that’s scheduled for the next meeting in Sept. Then we’d have a vote on the deterministic profile. In the context of relaxed SIMD, all the spec tests, texts exist and the requirements have been satisfied. But we need to have that one final  discussion on the profiles proposal, and have a vote for making sure that we have a deterministic profile that exists, which the relaxed SIMD spec depends on before we can advance to Phase 4.
+
+#### Teaser: a DSL for authoring the spec [Andreas Rossberg, 30 min]
+
+AR Presenting [TODO: Add slides]
```
Comment (dschuff) [inline main/2023/CG-09-12.md:189]: @rossberg TODO: spec DSL teaser slides

File: main/2023/CG-09-12.md (hunk 1 above)
Comment (rossberg) [reply main/2023/CG-09-12.md:118]: Sorry, please see #1427.

File: main/2023/CG-09-12.md (hunk 2 above)
Comment (rossberg) [reply main/2023/CG-09-12.md:189]: Please see #1427.

**Pull request merged**